xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -1424,7 +1424,7 @@ bool CJSONServiceDescription::prepareDescription(std::string &description, CVari
   }
 
   if (description.at(0) != '{')
-    description = StringUtils::Format("{{{:s}}}", description);
+    description = StringUtils::Format("{{{:s}}}", description.c_str());
```

Comment:
See the comment above. The new formatting library / implementation (`{fmt}`) is type-safe and can handle std::string.